src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball = lottery.getRandomBall();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        System.out.println(ball);
+        System.out.println(ball1);
+        System.out.println(ball2);
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/1b208eca-45dc-409e-8381-3e85dce066d7)
Use loop for this

Reply:
not fixed